packages/ui/src/proposals/modals/AddNewProposal/helpers.ts

```diff
@@ -172,7 +172,8 @@ export const schemaFactory = (api?: ProxyApi) => {
       isDiscussionClosed: Yup.boolean(),
       discussionWhitelist: Yup.array().when('isDiscussionClosed', {
         is: true,
-        then: Yup.array().required('Field is required'),
+        then: Yup.array().test(maxContext('The maximum block number is ${20}', 'maxTriggerBlock', false))
```

Comment:
It may be something like [here](https://github.com/Joystream/pioneer/blob/ab7b03cd9b068e01e9f383b0a0c89dd9fd8d402e/packages/ui/src/bounty/modals/AddBountyModal/helpers.ts#L120) - @WRadoslaw Please advise.
```suggestion
        then: Yup.array().test(maxContext('The whitelist is limited to ${max} members', (value, context) => {
        if (!value?.length) return true
        const validationContext = context.options.context as Conditions
        return (validationContext.maxWhitelistSize ?? BN_ZERO).gten(value.length)
      }), false))
```